astropy/io/ascii/core.py

```diff
@@ -652,10 +652,10 @@ def check_column_names(self, names, strict_names, guessing):
             # Impose strict requirements on column names (normally used in guessing)
             bads = [" ", ",", "|", "\t", "'", '"']
             for name in self.colnames:
-                if (_is_number(name) or
-                    len(name) == 0 or
-                    name[0] in bads or
-                    name[-1] in bads):
+                if ((_is_number(name) or
+                     len(name) == 0 or
+                     name[0] in bads or
+                     name[-1] in bads)):
```

Comment:
We generally try to keep the PRs focussed on what's actually changed. That can be hard to find if there are many lines that have just white spaces added and removed - specifically in a case where it's based on the preference of a linter and not on PEP8 itself. If you are willing to take up the work to tidy up the io.ascii module, then I'd happily review a PR that does just that (preferably for all files in that module).
And in this particular case, the indentation seems totally fine to me, if you remove the second pair  of brackets? Did you add another "(..)", just so that the indentation is *not* a multiple of four? I don't know if I've seen that in other parts of astropy before. @taldcroft what do you think?